ckanext/ontario_theme/templates/internal/organization/snippets/organization_item.html

```diff
@@ -1,16 +1,22 @@
 {% ckan_extends %}
-
-{% block title %}
-  <h3 class="media-heading">{{ h.get_translated(organization, "title") or organization.name }}</h3>
-{% endblock %}
-{% block description %}
-  {% if h.get_translated(organization,"description") %}
-    <p>{{ h.markdown_extract(h.get_translated(organization,"description"), extract_length=80) }}</p>
-  {% endif %}
-{% endblock %}
-{% block link %}
-  {% set organization_name = (h.get_translated(organization, "title") or organization.name) %}
-  <a href="{{ url }}" title="{{ _('View {organization_name}').format(organization_name=organization_name) }}" class="media-view">
-    <span>{{ _('View {organization_name}').format(organization_name=organization_name) }}</span>
-  </a>
+{% block item %}
+<li class="media-item ministry-item">
+  {% block item_inner %}
+    {% block image %}
+    {% endblock %}
+    {% block title %}
+      {% set organization_name = (h.get_translated(organization, "title") or organization.name) %}
+      <a href="{{ url }}" title="{{ _('View {organization_name}').format(organization_name=organization_name) }}" class="ministry-heading">
```

Comment:
I think this text is for a popup that displays when you hover over a ministry name. However, I don't see this pop-up on my local. Can you check?

Reply:
![Screenshot from 2023-01-03 11-54-16](https://user-images.githubusercontent.com/60950452/210403580-3a7d669f-480c-412d-a24b-30d0d9c1802b.png)

Reply:
I can see it now too